lab04_example/include/lab04_example/VelocityController.h

```diff
@@ -0,0 +1,73 @@
+/* Author: Luke Dennis l.dennis@unsw.edu.au */
+
+#include <ros/ros.h>
+#include <std_msgs/Int8.h>
+#include <geometry_msgs/Twist.h>
+#include <lab04_example/key_control.h>
+#include <lab04_example/KeyToCmdVel.h>
+
+namespace Lab04 {
+    /* Inspiration from:
+     *  https://github.com/wsnewman/ros_class/blob/master/example_ros_class/src/example_ros_class.h
+     *  https://roboticsbackend.com/roscpp-timer-with-ros-publish-data-at-a-fixed-rate/
+     */
+    class VelocityController {
+        public:
```

Comment:
Needs comments about what this class will do, how it relates to the lab

Reply:
4f5e9f7b18aad97f02a4c2dc5ecaa2cec1f6e6ae

Reply:
4f5e9f7b18aad97f02a4c2dc5ecaa2cec1f6e6ae